packages/webawesome/docs/assets/styles/utils.css

```diff
@@ -122,6 +122,29 @@
       --secondary-color: var(--wa-color-neutral-30);
     }
   }
+
+  /* anchor headings  */
+  .anchor-heading a {
+    opacity: 0;
+    visibility: hidden;
+    text-decoration: none;
+    transition: opacity var(--wa-transition-normal) var(--wa-transition-easing);
+  }
+
+  @media (hover: hover) {
+    .anchor-heading:hover a {
+      opacity: 1;
+      visibility: visible;
+      padding: var(--wa-space-3xs);
+    }
+  }
+
+  @media print {
```

Comment:
@lindsaym-fa fair to more this print-based styling here for all of the site to use? Or should we only do this for Docs-based content?